freedomofpress/securedrop.org
# Ensure that log in link is hidden if directory tools are disabled

Also fix some other footer template issues in the process

To test:

* Toggle settings in the wagtail admin under Settings > Directory Settings > Allow Directory Management
* Observe that the footer "Log In"/"Manage your SecureDrop directory listing" link toggles on and off with the setting (this link is on every page)

Resolves #430
